documentation/src/main/asciidoc/topics/marshalling.adoc

```diff
@@ -123,6 +93,73 @@ JBoss Marshalling is deprecated and planned for removal in a future version of {
 interface when persisting data unless you configure JBoss marshalling. However, this interface is also deprecated and planned for removal.
 ====
 
+=== Kryo Marshalling
+{brandname} provides a marshalling implementation that uses Kryo libraries.
```

Comment:
It could be worth adding a link to the Kryo and ProtoStuff libraries as part of the docs?

Reply:
do you mean the "bundle" JAR?

Reply:
disregard the previous question. I'm an idiot. updated with links to kryo and protostuff repos.